Sources/CasePaths/CasePathable.swift

```diff
@@ -440,15 +440,10 @@ extension CasePathable {
   /// userActions.filter { $0.is(\.home) }      // [UserAction.home(.onAppear)]
   /// userActions.filter { $0.is(\.settings) }  // [UserAction.settings(.subscribeButtonTapped)]
   /// ```
-  @_disfavoredOverload
   public func `is`(_ keyPath: PartialCaseKeyPath<Self>) -> Bool {
     self[case: keyPath] != nil
   }
 
-  public func `is`<Wrapped>(_ keyPath: CaseKeyPath<Self, Wrapped?>) -> Bool {
-    self[case: keyPath] != nil
-  }
```

Comment:
Unrelatedly, we found that this overload is no longer necessary with some upstream optional promotion fixes.